gpkit/fmincon/simpleflight.py

```diff
@@ -0,0 +1,62 @@
+"Minimizes airplane drag for a simple drag and structure model."
```

Comment:
Doesn't it require an `__init__` in every level above the `examples` directory though? And isn't that gross?